src/sonic_ax_impl/mibs/__init__.py

```diff
@@ -536,14 +519,35 @@ def get_oidvalue(self, oid):
         return self.oid_map[oid]
 
 class Namespace:
+
+    @staticmethod
+    def init_sonic_db_config():
```

Comment:
>init_sonic_db_config [](http://example.com/codeflow?start=8&length=20)

We could add a static member `Namespace.db_config_loaded`.
1. init_sonic_db_config will check it and only load once
2. init_db will call init_sonic_db_config
3. init_namespace_dbs will call init_sonic_db_config
4. other caller do not need to call init_sonic_db_config explicitly

Reply:
Fixed

Reply:
Sorry for misleading, I mean a private member boolean variable Namespace.db_config_loaded. It will be more efficient.

Reply:
Fixed, add a static class data member db_config_loaded